kod/object/active/holder/room/ghall.kod

```diff
@@ -1355,20 +1354,17 @@ messages:
    }  
 
    ReqLegalEntry(who=$)
+   "Returns TRUE if the player is in the guild hall legally; otherwise."
    {
       local i;
-
+      
+      % Unowned guild halls are always open
       if poGuild_Owner = $
       {
          return TRUE;
       }
 
-      if poGuild_owner = Send(who,@GetGuild)
```

Comment:
I don't think this belongs here. It isn't needed and just because someone is in the guild doesn't mean they entered legally.

Reply:
Doesn't this make it impossible to take anything from the guild chest?  I think we should go back to a more minimal change that just fixes the allied exit bug.

Reply:
Yes, since it will now rely entirely on hotplates. I can reintroduce this logic here to allowlist guild members, even if they’re not on the legal entry list, or simply roll this back to reduce the complexity. I'll assume you're in favor of the latter and plan to roll it back.